strings_lab.js

```diff
@@ -0,0 +1,220 @@
+// // 1. `console.log` the first character of a string.
```

Comment:
Good understanding of string methods and properties. I see minor issue with indentation and line breaking but other than that no major feedback. Keep up the good work and keep doing bonus questions!